modules/custom/cu_core/cu_core.module

```diff
@@ -61,6 +61,9 @@ function cu_core_form_alter(&$form, $form_state, $form_id) {
   if (isset($form['type']) && isset($form['#node']) && $form['type']['#value'] .'_node_form' == $form_id) {
     unset($form['options']['promote']);
   }
+  if ($form['actions']['preview_changes']) {
```

Comment:
if you don't set as isset check first, checking for an undefined index will get logged as a notice.